config/samples/end-to-end.yaml

```diff
@@ -49,16 +37,12 @@ spec:
               }, "binet", null
             ]
           }
-        }
-      }
-    }
 ---
 # Deployment of a demo-app using our custom resource
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: open-feature-demo-deployment
-  namespace: open-feature-demo
```

Comment:
Removed the namespaces for now, related to https://github.com/open-feature/open-feature-operator/issues/201

Reply:
Does this need to be addressed before we merge?

Reply:
No, there is a separate issue addressing this